jenkins_jobs/trigger_weekly_host_os_build/pre_build_script.sh

```diff
@@ -22,5 +35,7 @@ python host_os.py \
 	--push-repo-url "ssh://git@github.com/${GITHUB_BOT_USER_NAME}/${GITHUB_ORGANIZATION_NAME}.github.io.git" \
 	--push-repo-branch "$COMMIT_BRANCH"
 
+create_pr "${GITHUB_ORGANIZATION_NAME}.github.io"
+
 echo "VERSIONS_REPO_COMMIT=$COMMIT_BRANCH" > BUILD_PARAMETERS
 echo "$RELEASE_DATE" > WEEKLY_DIR_NAME
```

Comment:
Well, since the .sh script is fairly simple, I would like to skip the extra level of indirection that a python script would create. It is much simpler for instance to understand that `echo "$RELEASE_DATE" > WEEKLY_DIR_NAME` creates a file than to read that from a script somewhere else. I think it makes it easier to manage and interact with the jobs if we have a little snippet of code to read. There's also the fact that we already have cmdline tools that do their work quite well (rsync, echo, date, bash parameter substitution) so I see no need to re-implement them in python. In general, I think we should keep what is related to Jenkins, jobs variable manipulation, job flow, inter-job communication (like BUILD_PARAMETERS) or things that can be accomplished with cmdline tools (like rsync) as shell script.

However, I do understand your point and that's why I wrote the github_api script instead of making a thread ball of curl and grep here.